### DISCUSSION TITLE
Which agent should we use for a hybrid deployment in GCP?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18208] [Q&A] [answered]
I'm considering a hybrid deployment in GCP; is the docker agent the right one to use? we use Cloud Run extensively, fwiw - so if there were something that integrated more closely with that, it would be interesting

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C02LJ7G0LAZ/p1700603603770299?thread_ts=1700603603.770299&cid=C02LJ7G0LAZ).*

**Comment by gibsondan** [comment, marked as answer]
Hi Venky! We don't have a Cloud Run agent currently, but we have many users using GKE with the Kubernetes agent. If that's not an option for whatever reason, then the Docker agent on a single node/instance is probably your best bet.

**Reply by indigoviolet** [reply]
Thanks Dan!
